### PR TITLE
Update the results, fix validations, and add omit empty

### PR DESCRIPTION
**Pull request by dstiliadis**
## Description

Fixes several issues in cloud graph:

1. Update validations to match all the protocols from AWS and fix bugs there.
2. Update validations and change the notations around native IDs.
3. Add omit empty in objects in the results so that we reduce bandwidth
4. Add indexes for optimization.

### CODE REVIEW

File: specs/cloudnode.spec [hunk 1]
```diff
@@ -19,6 +19,8 @@ model:
 
 # Indexes
 indexes:
+- - namespace
```
Comment (primalmotion) [inline specs/cloudnode.spec:22]: This is unnecessary. The following index covers this one

**Review by primalmotion — CHANGES_REQUESTED** [review]

File: specs/cloudnode.spec (hunk 1 above)
Comment (dstiliadis) [reply specs/cloudnode.spec]: Please check the new indexes. This is what gave the best performance.

**Review by primalmotion — APPROVED** [review]

**Comment by dstiliadis** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "graph-v2",
    "component": "gaia",
    "pr-id": "627",
    "commit-sha": "5aadc409fb7d8b427429df09266f50d5d5cf39c7",
    "pipeline": "master"
  },
  {
    "project": "graph-v2",
    "component": "backend",
    "pr-id": "1103",
    "commit-sha": "e8b213295fb14e6109a738e7fd894186963d8a60",
    "pipeline": "master"
  }
]
```

**Comment by dstiliadis** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "graph-v2",
    "component": "gaia",
    "pr-id": "627",
    "commit-sha": "5aadc409fb7d8b427429df09266f50d5d5cf39c7",
    "pipeline": "master"
  },
  {
    "project": "graph-v2",
    "component": "backend",
    "pr-id": "1103",
    "commit-sha": "48aaf9c5ddff18ec7966a2cbf1927edf68d16cef",
    "pipeline": "master"
  }
]
```

**Review by primalmotion — APPROVED** [review]

**Pull request merged**